SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/Storage/Storage.cs

```diff
@@ -298,13 +279,13 @@ internal Storage(IStorageAdapter storageAdapter)
                 logs?.AddRange(retrievedLogs);
                 return batchId;
             }
-            catch (DbException e)
+            catch (StorageException)
             {
-                throw new StorageException("Error retrieving logs from storage", e);
+                throw new StorageException("Error retrieving logs from storage");
```

Comment:
>throw new StorageException("Error retrieving logs from storage"); [](start = 16, length = 65)

Catch and re-throw the same storage exception with all internal information stripped off. Can we keep `throw new StorageException("Error retrieving logs from storage", e);` here?